src/interactive/Dropdown.jsx

```diff
@@ -77,6 +78,12 @@ class Dropdown extends React.PureComponent {
 		}));
 	}
 
+	scheduleContentPosition(triggerClientRect) {
```

Comment:
why not call `scheduleUpdate` directly?

Reply:
Because then I can't clean it up on unmount with `this.scheduleUpdate.cancel();`